pkg/mutation/mutator.go

```diff
@@ -61,3 +66,19 @@ func MakeID(obj runtime.Object) (ID, error) {
 		Namespace: meta.GetNamespace(),
 	}, nil
 }
+
+// MutatorFor returns a mutator generated
```

Comment:
I think we removed the MutatorFor function after refactoring the MutationSystem interface

Reply:
Yes, was a bit torn about having a single entry point that dispatches to the right function or two type safe ones.
Will remove it